020-Getting-started/020-installation.mdx

```diff
@@ -89,14 +89,48 @@ export const xata = new XataClient({ apiKey: process.env.XATA_API_KEY })
 from xata.client import XataClient
 client = XataClient()
 ```
+```go title="utils/xata.go"
+import (
+	"github.com/xata-api/go/xata"
+)
+func main() {
+	// Environment variables, like `XATA_API_KEY`, are detected from the client
+	client := xata.NewClient()
+}
+```
 </TabbedCode>
 
+### Committing files to your repository
+
+For collaborative development in a common Workspace it is important to set up your local environment so that it can be replicated by others. Commit these specific files to your project's repository:
+
+- `.xata`: This folder holds the migration files, managing changes to the database schema over time.
+- `.xatarc`: This file contains essential configuration information for your database setup.
+- `src/xata.ts`: Includes the Xata client and API integration, responsible for interacting with the Xata service from code.
+
+Committing these files to your project's repository, enables Workspace members to replicate your local setup by adding their own `XATA_API_KEY` to their `.env` file.
+
+To deploy your GitHub repository on hosting services like Vercel or Netlify, ensure you include the `src/client.ts` file from the Xata client in your repo. Not exposing your API key, ensures your Xata database is safe from DDoS attacks, even if the URL is visible. Remember, requests made without authentication won’t impact your Xata database's [rate limits](/docs/concepts/pricing#concurrency-limit). Exposing the `src/client.ts` file doesn't pose a risk to your database’s limits or security.
+
+When creating templates or examples for public use and cloning, it is best to avoid committing files that contain specific database API endpoint and client files. Configuration files such as `src/xata.ts`,`.xatarc` and the `.xata` migrations folder are tied to a specific database.
```

Comment:
```suggestion
When creating templates or examples for public use and cloning, it is best to avoid committing files that contain specific database API endpoint and client files. Configuration files such as `src/xata.ts`,`.xatarc`, and the `.xata` migrations folder are tied to a specific database.
```